app/bundles/LeadBundle/Views/List/form.html.php

```diff
@@ -119,6 +119,11 @@ class="segment-filter <?php echo $icon; ?>">
                             <div class="clearfix"></div>
                         </div>
                         <div class="selected-filters" id="leadlist_filters">
+                            <?php if ($filterErrors): ?>
+                                <div class="alert alert-danger">
```

Comment:
Add the css class `has-error` before `alert` and it should style like any other red error box in Mautic.

Reply:
Thanks. I've add it.